src/utils/formatting.ts

```diff
@@ -9,7 +9,8 @@ export function normalizeToKebabOrSnakeCase(str: string) {
   const STRING_DASHERIZE_REGEXP = /\s/g;
   const STRING_DECAMELIZE_REGEXP = /([a-z\d])([A-Z])/g;
   return str
-    .replace(STRING_DECAMELIZE_REGEXP, '$1-$2')
-    .toLowerCase()
-    .replace(STRING_DASHERIZE_REGEXP, '-');
+    ?.trim()
+    ?.replace(STRING_DECAMELIZE_REGEXP, '$1-$2')
+    ?.toLowerCase()
+    ?.replace(STRING_DASHERIZE_REGEXP, '-');
```

Comment:
are we sure that the `str` parameter can be nil? :thinking:

Reply:
> are we sure that the `str` parameter can be nil? 🤔 if yes, please add yet another test to cover this scenario

So I just checked into the source and I saw that all implements with default value ...but I just added more unit tests